### DISCUSSION TITLE
How to route to /foo/

### DISCUSSION THREAD

**Discussion by rbtcollins** [tokio-rs/axum#2775] [Q&A] [answered]
### Summary

I have an API that I'm trying to match - I don't control the clients, and I'm quite confused by the various discussion in #2659 #1118 #714 ...

what is the official Right Way to have a route like

```
/track/ -> some handler
```

?

### axum version

0.6.12

**Comment by rbtcollins** [comment, marked as answer]
And I found I had simply changed a test instance not the main router. Mea culpa! It just works in the obvious way.

**Reply by mladedav** [reply]
I'll just add that using `route` should just work, issues usually arise when someone tries to use `nest` because there a trailing slash matters for the nested root path.
